.github/workflows/build-and-test.yml

```diff
@@ -69,32 +69,18 @@ jobs:
       - name: Install tools
         if: steps.tool-cache.outputs.cache-hit != 'true'
         run: make install-tools
-      - name: split loadtest jobs
+      - name: Split Loadtest Jobs
         id: splitloadtest
-        run: |
-          TESTS="$(make -s testbed-list-loadtest | xargs echo|sed 's/ /|/g')"
-          TESTS=(${TESTS//|/ })
-          MATRIX="{\"include\":["
-          curr=""
-          for i in "${!TESTS[@]}"; do
-            if (( i > 0 && i % 2 == 0 )); then
-              curr+="|${TESTS[$i]}"
-            else
-              if [ -n "$curr" ] && (( i>1 )); then
-                MATRIX+=",{\"test\":\"$curr\"}"
-              elif [ -n "$curr" ]; then
-                MATRIX+="{\"test\":\"$curr\"}"
-              fi
-              curr="${TESTS[$i]}"
-            fi
-          done
-          MATRIX+="]}"
-          echo "::set-output name=matrix::$MATRIX"
-      - name: Upload tool binaries
+        run: ./.github/workflows/scripts/setup_load_tests.sh
+      - name: Add Execute Permissions to Binaries
+        run: chmod -R +x /home/runner/go/bin
+      - name: Create Tool Binaries Archive
+        run: tar -cvf tool-bin.tar /home/runner/go/bin
```

Comment:
why is this needed vs the whole path?

Reply:
The reason I archived the file was to preserve the execute permissions on the binaries created. Specifically on the windows-msi job, there was no way to easily `chmod +x` the binary once downloaded given the OS. Also, this prevents having to add permissions to the binaries everytime it is downloaded although extracting the archive is required instead.

Reply:
I am fine to always add the chmod +x instead of archive. Maybe leave also a TODO to explain this.

Reply:
The only issue is that I cannot do the chmod for the windows job (In PR 3/3), this was a workaround specifically for that. I can remove the archiving from tool binaries, but it is needed for the collector binaries which the windows-msi job uses.

Reply:
Removed the archiving from all the tool binaries, because it was unnecessary. I still need to archive the collector binary produced from cross-compile because it requires execute permissions to run in `windows-msi`.

Reply:
It does not seem to be reflected in the code.

Reply:
I pushed my amend to the wrong Git branch, did not mean to mislead sorry. All instances of tool archive should be gone now.